assets/src/edit-story/app/story/storyProvider.js

```diff
@@ -71,6 +75,8 @@ function StoryProvider( { storyId, children } ) {
 			setCurrentPageByIndex,
 			addBlankPage,
 			clearSelection,
+			deleteSelectedElements,
```

Comment:
There's a similar question with history work, but maybe it's worth doing reducer/dispatch for all datamodel updates? Totally ok deferring this question until further pull requests. Just brining attention to whether the APIs are starting to grow pretty wide.

Reply:
Probably better to discuss this under a more general issue. If that's OK for you then I'd prefer merging the PR and later changing together with the other actions if needed. Thoughts?

Reply:
I think that using `reducer/dispatch` is a larger architectural discussion as other part of this implementation would have to be changed. @dvoytenko can you create another ticket for this so we can move the discussion there?

Reply:
I created a new "Definition" story in https://github.com/ampproject/amp-wp/issues/3894. LGTM to this PR.